src/tools/rest-frontend/resources/assets/js/controllers/main/MainController.js

```diff
@@ -12,7 +12,7 @@ module.exports = function ($rootScope, $scope, Logger, $state, $anchorScroll, we
 
     // build date
     $scope.builddate = {
-        pretty: (new Date(document.querySelector('meta[name="build-date"]').getAttribute('content'))).toLocaleString()
+        pretty: (new Date(document.querySelector('meta[name="build-date"]').getAttribute('content'))).toUTCString()
```

Comment:
The local one is actually a nice feature, there should be only some information which "local" one was chosen.